diff --git a/libs2e/CMakeLists.txt b/libs2e/CMakeLists.txt
index 1bec5da..a8b9fe1 100644
--- a/libs2e/CMakeLists.txt
+++ b/libs2e/CMakeLists.txt
@@ -86,6 +86,12 @@ message(STATUS "Found libvmi ${VMI_PACKAGE_VERSION}")
 find_package(LIBCOROUTINE REQUIRED)
 message(STATUS "Found libcoroutine ${LIBCOROUTINE_PACKAGE_VERSION}")
 
+find_package(Python COMPONENTS Interpreter Development REQUIRED)
+message(STATUS "Found Python ${PYTHON_PACKAGE_VERSION}")
+
+find_package(pybind11 REQUIRED)
+message(STATUS "Found pybind11")
+
 if(WITH_TARGET MATCHES "s2e")
     # TODO: look at libcpu compile options to figure this out
     find_package(LIBS2ECORE REQUIRED)
diff --git a/libs2e/src/CMakeLists.txt b/libs2e/src/CMakeLists.txt
index 6ababfd..aa00d40 100644
--- a/libs2e/src/CMakeLists.txt
+++ b/libs2e/src/CMakeLists.txt
@@ -64,7 +64,7 @@ if(WITH_TARGET MATCHES "s2e")
                      boost_regex)
 endif()
 
-set(LIBS ${LIBS} ${LIBCOROUTINE_LIBRARY_DIR}/libcoroutine.a ${FSIGCXX_LIBRARY_DIR}/libfsigc++.a ${CAPSTONE_LIB_DIR}/libcapstone.a pthread glib-2.0 bsd)
+set(LIBS ${LIBS} ${LIBCOROUTINE_LIBRARY_DIR}/libcoroutine.a ${FSIGCXX_LIBRARY_DIR}/libfsigc++.a ${CAPSTONE_LIB_DIR}/libcapstone.a pthread glib-2.0 bsd pybind11::embed)
 
 target_link_libraries(tests2e PUBLIC ${LIBCOROUTINE_LIBRARY_DIR}/libcoroutine.a pthread glib-2.0)
 target_link_libraries(s2e ${LIBS})
