diff --git a/libs2eplugins/CMakeLists.txt b/libs2eplugins/CMakeLists.txt
index 933ae3e..087d621 100644
--- a/libs2eplugins/CMakeLists.txt
+++ b/libs2eplugins/CMakeLists.txt
@@ -100,6 +100,12 @@ message(STATUS "Found s2e core ${LIBS2ECORE_PACKAGE_VERSION}")
 find_package(KLEE REQUIRED)
 message(STATUS "Found klee ${KLEE_PACKAGE_VERSION}")
 
+find_package(Python COMPONENTS Interpreter Development REQUIRED)
+message(STATUS "Found Python ${PYTHON_PACKAGE_VERSION}")
+
+find_package(pybind11 REQUIRED)
+message(STATUS "Found pybind11")
+
 include_directories(${GLIB_PKG_INCLUDE_DIRS}
                     ${LIBQ_INCLUDE_DIR}
                     ${LIBTCG_INCLUDE_DIR}
@@ -110,6 +116,8 @@ include_directories(${GLIB_PKG_INCLUDE_DIRS}
                     ${FSIGCXX_INCLUDE_DIR}
                     ${KLEE_INCLUDE_DIR}
                     ${LUA_DIR}
+                    ${PYTHON_INCLUDE_DIR}
+                    ${PYBIND11_INCLUDE_DIR}
                     ${S2EGUEST_INCLUDE_DIR})
 
 add_subdirectory(src)
